packages/cardpay-sdk/sdk/utils/conversions.ts

```diff
@@ -123,3 +124,21 @@ export async function getNativeWeiInToken(provider: JsonRpcProvider, tokenAddres
   let rateAdjusted = adjustRate(rate);
   return new BN(rateAdjusted.numerator.toString()).div(new BN(rateAdjusted.denominator.toString()));
 }
+
+export async function getUsdConverter(
```

Comment:
The output of this is the conversion rate, right? Let's rename to `getUsdConversionRate`

Reply:
The output is a function to convert token amount to usd amount.

Reply:
Ok, I see